src/ResourceManager/Kusto/Commands.Kusto.Test/Commands.Kusto.Test.Netcore.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(ProjectDir)..\..\..\..\tools\Common.Netcore.Dependencies.Test.targets" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp2.0</TargetFrameworks>
+    <AssemblyName>Microsoft.Azure.Commands.Kusto.Test</AssemblyName>
```

Comment:
Please use `Microsoft.Azure.PowerShell.Cmdlets.Kusto.Test` as the `AssemblyName`